packages/cli/src/commands/preview-docs/index.ts

```diff
@@ -136,7 +136,7 @@ export async function previewDocs(argv: {
 
     const redoclyClient = new RedoclyClient();
     isAuthorizedWithRedocly = await redoclyClient.isAuthorizedWithRedocly();
-    const referenceDocs = config.referenceDocs || {};
+    const referenceDocs = config['features.openapi'] || {};
```

Comment:
Can we use merged config here so `features.openapi` is used from the specific API ?

Reply:
Updated the code. Please see the commit: https://github.com/Redocly/openapi-cli/pull/588/commits/d95e9d89bb8d72cab75eb5804fdfe96666ccb20e